scanner/metadata_ffmpeg.go

```diff
@@ -269,23 +269,17 @@ func (m *Metadata) parseDuration(tagName string) float32 {
 }
 
 func createProbeCommand(inputs []string) (string, []string) {
-	cmd := conf.Server.ProbeCommand
-
-	split := strings.Split(cmd, " ")
 	args := make([]string, 0)
-	first := true
-	for _, s := range split {
-		if s == "%s" {
-			for _, inp := range inputs {
-				if !first {
-					args = append(args, "-i")
-				}
-				args = append(args, inp)
-				first = false
-			}
+	args = append(args, conf.Server.ProbeCommand...)
+	args = append(args, "-i")
```

Comment:
See above about duplicating the `-i`

Also, `ffmpeg` requires one `-i` for each input file.